components/service/glean/src/test/java/mozilla/components/service/glean/DispatchersTest.kt

```diff
@@ -5,33 +5,29 @@
 package mozilla.components.service.glean
 
 import kotlinx.coroutines.runBlocking
-import org.junit.Test
-
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertNotSame
 import org.junit.Assert.assertSame
+import org.junit.Test
 
 class DispatchersTest {
 
     @Test
-    fun `API scope runs off the main thread`() {
+    @Suppress("EXPERIMENTAL_API_USAGE")
+    fun `API scope runs off the main thread`() = runBlocking {
```

Comment:
Since we're moving things to a separate repo, we're in the process of keeping things in sync. Unless test changes are strictly required to use the `androidx.runner`, could you consider dropping non-essential changes to `service-glean`? We already have some diverging test code in [mozilla/glean](https://github.com/mozilla/glean) and we'd have some hard time keeping things in sync otherwise. We'll gladly fix things there too, once we reach "feature parity". Before then, changing tests might lead to unpredictable behaviour changes (even though I understand that the changes in this PR *shouldn't*).

Thanks for your great efforts on this anyway, code looks much cleaner!